dts/common/nordic/nrf54h20.dtsi

```diff
@@ -168,6 +168,20 @@
 				interrupts = <64 NRF_DEFAULT_IRQ_PRIORITY>,
 					     <65 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			cpuapp_wdt010: watchdog@14000 {
+				compatible = "nordic,nrf-wdt";
+				reg = <0x14000 0x1000>;
```

Comment:
Add also corresponding entries in `validate_base_addresses.c`.

Reply:
This is a black magic for me.
I found following defines in HAL
modules/hal/nordic:
nrfx/mdk/nrf54h20_radiocore.h:    #define NRF_WDT010                            NRF_RADIOCORE_WDT010
nrfx/mdk/nrf54h20_radiocore.h:    #define NRF_WDT010                            NRF_RADIOCORE_WDT010
nrfx/mdk/nrf54h20_application.h:    #define NRF_WDT010                            NRF_APPLICATION_WDT010
nrfx/mdk/nrf54h20_application.h:    #define NRF_WDT010                            NRF_APPLICATION_WDT010

Reply:
I think we should use `CHECK_DT_REG(cpuapp_wdt010, NRF_APPLICATION_WDT010);` and so on.

Reply:
Done. Please confirm it's all what's needed.

Reply:
LGTM now, thanks.